.github/workflows/lint.yml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022 The StableHLO Authors.
+
+name: Lint
+
+on: [pull_request]
+
+jobs:
+  # Based on https://github.com/iree-org/iree/blob/main/.github/workflows/lint.yml
+  clang-format:
+    if: "github.event_name == 'pull_request'"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Installing dependencies
+        run: |
+          wget https://raw.githubusercontent.com/llvm-mirror/clang/master/tools/clang-format/git-clang-format -O /tmp/git-clang-format
+          chmod +x /tmp/git-clang-format
+      - name: Checking out repository
+        uses: actions/checkout@v2
+      - name: Fetching Base Branch
+        # We have to explicitly fetch the base branch as well
+        run: git fetch --no-tags --prune --depth=1 origin "${GITHUB_BASE_REF?}:${GITHUB_BASE_REF?}"
+      - name: Running clang-format on changed source files
+        run: |
+          /tmp/git-clang-format "${GITHUB_BASE_REF?}" --style=google
+          git diff --exit-code
```

Comment:
Would it be possible to also run `clang-tidy`, bootstrapping the config from MLIR-HLO to begin with? https://github.com/tensorflow/mlir-hlo/blob/master/.clang-tidy. Doesn't have to block this PR.

Reply:
Created #61